README.md

```diff
@@ -1,11 +1,78 @@
 RateLimit.js
 ============
 [![Build
-Status](https://travis-ci.org/dudleycarr/ratelimit.js.svg)](https://travis-ci.org/dudleycarr/ratelimit.js)
+Status](https://travis-ci.org/dudleycarr/ratelimit.js.svg)](https://travis-ci.org/dudleycarr/ratelimit.js) [![npm version](https://badge.fury.io/js/ratelimit.js.svg)](http://badge.fury.io/js/ratelimit.js)
 
-A NodeJS library for rate limiting using sliding windows stored in Redis.
+A NodeJS library for efficiently rate limiting using sliding windows stored in Redis.
```

Comment:
Minor nitpick, but I think "A nodeJS library for efficient rate limiting using sliding windows..." reads a bit more naturally :smile: